salsa_dancing_molecules/mean_square_displacement.py

```diff
@@ -0,0 +1,89 @@
+"""Module for calculating the mean square displacement of a trajectory file."""
+
+from ase.io import read
+import numpy as np
+
+
+def get_msd(traj_file, reference="final"):
+    """
+    Read trajectory file and call calculation function.
+
+    arguments:
+        traj_file - trajectory file
+        reference - string or None
+
+    return:
+        mean square displacement - list
+
+    Example of a function call:
+        get_msd(traj_file = "cu.traj@0:-1")
```

Comment:
I think an explanation as to what the @0:-1 of the file name means would be great in the documentation comment.

Reply:
What comment is the documentation comment?

Reply:
The documentation comment is the comment where I put my Github comment, i.e. the comment where you explain what the function does.